roles/cinder-data/templates/etc/cinder/cinder.encryption.conf

```diff
@@ -1,4 +1,7 @@
-[keymgr]
-# This line is needed, but we do not actually want
-# a real key saved on the cinder nodes, hence 'NOT_A_KEY'
-fixed_key = NOT_A_KEY
+[key_manager]
```

Comment:
This seems to be another duplicate file? I guess there are multiple places we can place this?